File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int min = 1000000, max = -1000000;
        int[] array = new int[n];
        for(int i = 0; i < n; i++){
            array[i] = in.nextInt();
            if(min > array[i])
                min = array[i];
            if(max < array[i])
                max = array[i];
        }
        System.out.printf("%d %d", min, max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var big = -1000000; var sma = 1000000
    for(i in 1..sc.nextInt()){
        var n = sc.nextInt()
        if(big < n)
        big = n
        if(sma > n)
        sma = n
    }
    print("$sma $big")
}